build-system/tasks/test-report-upload.js

```diff
@@ -72,6 +72,7 @@ function addJobAndBuildInfo(testType, reportJson) {
   }
 
   return {
+    repository: process.env.GITHUB_REPOSITORY,
```

Comment:
In addition to this, WDYT about gating line 133 on `isTravisBuild()` to prevent duplicate uploads when we start testing CircleCI builds with `ampproject/amphtml` PRs? (We can remove the gate when Travis runs are shut down.)

Reply:
My inclination would be that CircleCi builds should instead not pass the `--report` flag when calling the gulp tasks in `dist-tests.js`, `e2e-tests.js`, and `local-tests.js` and the test-report-upload task itself shouldn't do the gating. WDYT?

Reply:
That works. Will do.